From 4af81a96371b1be21845dc9403ce4175f24b39ff Mon Sep 17 00:00:00 2001
Message-Id: <4af81a96371b1be21845dc9403ce4175f24b39ff.1449240176.git.mprivozn@redhat.com>
From: Michal Privoznik <mprivozn@redhat.com>
Date: Fri, 10 Jul 2015 10:49:27 +0200
Subject: [PATCH] LIBVIRT-MIB: Comply with formatting rules
MIME-Version: 1.0
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: 8bit

https://bugzilla.redhat.com/show_bug.cgi?id=1242320

So, I've came along this web site [1] which tries to validate a MIB
file. So I fed it with LIBVIRT-MIB and found some errors. This commit
fixes them so that the test passes again. This should be no functional
change, rather than some formatting change.

1: http://www.simpleweb.org/ietf/mibs/validate/

Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
(cherry picked from commit 200998484807a3d86f34bc5dab18bd5a8d3e2584)
Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
---
 src/LIBVIRT-MIB.txt | 14 ++++++--------
 1 file changed, 6 insertions(+), 8 deletions(-)

diff --git a/src/LIBVIRT-MIB.txt b/src/LIBVIRT-MIB.txt
index 932dec9..88c8afa 100644
--- a/src/LIBVIRT-MIB.txt
+++ b/src/LIBVIRT-MIB.txt
@@ -6,12 +6,10 @@ LIBVIRT-MIB DEFINITIONS ::= BEGIN
 
 IMPORTS
     MODULE-IDENTITY, OBJECT-TYPE, enterprises, Unsigned32, Gauge32,
-    Counter64
-	FROM SNMPv2-SMI
-    OBJECT-GROUP, MODULE-COMPLIANCE
-        FROM SNMPv2-CONF
-    TEXTUAL-CONVENTION, RowStatus
-	FROM SNMPv2-TC
+    Counter64 FROM SNMPv2-SMI
+    OBJECT-GROUP, MODULE-COMPLIANCE FROM SNMPv2-CONF
+    TEXTUAL-CONVENTION, RowStatus FROM SNMPv2-TC
+    NOTIFICATION-TYPE FROM SNMPv2-SMI
 ;
 
 
@@ -78,7 +76,7 @@ LibvirtGuestEntry ::= SEQUENCE {
 
 libvirtGuestUUID OBJECT-TYPE
     SYNTAX      UUID
-    MAX-ACCESS  not-accessible
+    MAX-ACCESS  accessible-for-notify
     STATUS      current
     DESCRIPTION
 	"The UUID of the virtual guest."
@@ -202,11 +200,11 @@ libvirtGuestGroup OBJECT-GROUP
     ::= { libvirtGroups 1 }
 
 libvirtGuestNotif NOTIFICATION-TYPE
-    STATUS current
     OBJECTS { libvirtGuestName,
               libvirtGuestUUID,
               libvirtGuestState,
               libvirtGuestRowStatus }
+    STATUS current
     DESCRIPTION
         "Guest lifecycle notification."
     ::= { libvirtNotifications 1 }
-- 
1.8.3.1

